### PR TITLE
Add The Canberra Times, Le Devoir and The Hindu + solve a NYT popup issue + popup.html redesign & others

### PR DESCRIPTION
**Pull request by iscreamcoke**
1. Add canberratimes.com.au (in collab. with @magnolia1234), ledevoir.com, thehindu.com

2. Add a 'return' button linking options.html to popup.html

3. Solve the issue in case of NYT popup "This is your one article preview".

4. popup.html redesign: "GitHub" renamed "Homepage"; redirect to readme.md

5. Reorder two websites according to their official name in readme.md & options.js (Zeit Online > [Die Zeit](https://en.wikipedia.org/wiki/Die_Zeit) ; Irish Times > [The Irish Times](https://en.wikipedia.org/wiki/The_Irish_Times)) & other maintenance tasks in lists

### CODE REVIEW

**Comment by magnolia1234** [comment]
You've have been busy again ;-)

For The Hindu instead the banner you can remove the parent-div with id=test (contains also paywall-script).

**Comment by iscreamcoke** [comment]
Done thanks! Well it's a good way to improve my basic JS skills, as I'm not a professional dev

Would you be interested in adapting a JS code for various Brazilian websites (or just the three biggest ones: O Globo; O Estado de S. Paulo; Folha de S.Paulo) from [this repo](https://github.com/burlesco/burlesco/)? (it's MIT-licenced but I'll ask the devs out of courtesy).

In general, I'm trying to add the websites I'm able to bypass [from this list](https://en.wikipedia.org/wiki/Newspaper_of_record), as these are the ones which are the most likely to be requested.

**Comment by iamadamdev** [comment]
We should try to avoid manually adding every site under the sun. We need a more generic solution.
A lot of sites can simply be bypassed by removing cookies, which is why I created https://chrome.google.com/webstore/detail/cookie-remover/kcgpggonjhmeaejebeoeomdlohicfhce
If the site can we bypassed in such a way, then lets not add it to Bypass Paywalls.
E.g., please remove darkreading.com since that can be bypassed by Cookie Remover (I haven't tried the others).

**Comment by iscreamcoke** [comment]
Well Dark Reading was requested but I've removed it. The other ones are very big papers in their respective country

**Comment by iscreamcoke** [comment]
Well Dark Reading was requested but I've removed it. The other ones are major papers in their respective country (Australia, Canada, India).

The Canberra Times needs a JS injection. Le Devoir and The Hindu can be bypassed by blocking cookies with your extension, then the pageview counter in uBlock for aesthetics. But let's be honest, most GitHub requests can be bypassed that way, I'm not sure the majority of users will know how to do it (i.e., blocking elements in uBlock). Do you want me to remove both of them anyway?

**Comment by magnolia1234** [comment]
Maybe an option to add a custom list of websites (someone suggested that before)?

**Comment by iscreamcoke** [comment]
We could merge the two extensions (bypassPaywall with a button in popup.html to block cookies for a website).

For banners and popups I had proposed to create a .txt of elements to be blocked, then optionally call it from background.js. But that’s imply maintenance on the long term.

**Comment by magnolia1234** [comment]
Like suggested in Firefox version:
https://github.com/iamadamdev/bypass-paywalls-firefox/issues/315
https://github.com/iamadamdev/bypass-paywalls-firefox/issues/88

**Comment by iscreamcoke** [comment]
Hadn’t see those, that’s exactly it. I guess that’s why Adam created the second extension in the first place, but I think it’s confusing for users to have two extensions.

File: options.html [hunk 1]
```diff
@@ -26,6 +26,7 @@
   <br/>
   <span style='float:left;'>
     <button id="save">Save</button>
+    <a href='popup.html'><button>Return</button></a>
```
Comment (iamadamdev) [inline options.html]: Not necessary

**Comment by iamadamdev** [comment]
Please fix merge conflicts

**Comment by iscreamcoke** [comment]
Fixed

File: popup.html [hunk 2]
```diff
@@ -1,8 +1,16 @@
 <!DOCTYPE html>
 <html>
+<head><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (iamadamdev) [inline popup.html]: I prefer not to rely on any third party css files. I'm going to merge this patch leaving popup.html unchanged.

**Comment by magnolia1234** [comment]
Btw Die Zeit and The Irish Times are no longer default options because their line in options.js differs from background.js.

**Comment by iscreamcoke** [comment]
Indeed. I had forgotten to do the change in background.js. It should be fixed by PR #395

**Pull request merged**